### PR TITLE
Add three-DC-in-two-cities deployment

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add three-DC-in-two-cities deployment doc.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3243 https://github.com/pingcap/docs-cn/pull/3829
- Other reference link(s):

### CODE REVIEW

File: three-data-centers-in-two-cities-deployment.md [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+  - host: 10.63.10.60
+```
+
+### Labels design
+
+In the deployment model of three DCs in two cities, the design of labels should have thought for availability and disaster recovery. It is recommended that you define the four levels (`dc`, `zone`, `rack`, `host`) based on the physical structure of the deployment.
+
+![Label logical definition](/media/three-data-centers-in-two-cities-deployment-03.png)
+
+PD 设置为 PD 设置 TiKV 部署位置等级信息。
```
Comment (TomShawn) [inline three-data-centers-in-two-cities-deployment.md:132]:
```suggestion
Add level information of TiKV labels in the PD configuration:
```

File: three-data-centers-in-two-cities-deployment.md [hunk 2]
```diff
@@ -0,0 +1,204 @@
+---
+title: Deployment Solution for Three Data Centers in Two Cities
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:2]:
```suggestion
title: Three Data Centers in Two Cities Deployment
```

File: three-data-centers-in-two-cities-deployment.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+summary: Learn the deployment solution for three data centers in two cities.
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:3]:
```suggestion
summary: Learn the deployment solution to three data centers in two cities.
```

File: three-data-centers-in-two-cities-deployment.md [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+category: how-to
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:4]:
```suggestion
category: tutorials
```

File: three-data-centers-in-two-cities-deployment.md [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+---
+
+# Deployment Solution for Three Data Centers in Two Cities
```
Comment (lilin90) [inline three-data-centers-in-two-cities-deployment.md:7]:
```suggestion
# Three Data Centers in Two Cities Deployment
```

**Comment by lilin90** [comment]
@freyfl PTAL

**Pull request merged**